### ISSUE TITLE
[Bug]: Problem with EAC3 7.1 & DTS-ES 6.1

### ISSUE THREAD

**Issue by mitch133** [nova-video-player/aos-AVP#727]
### Problem description

No audio and/or very poor playback experience since 6.0.75 with EAC3 7.1 files. No problems with EAC3 5.1 or other 7.1 codecs (like AAC)

### Steps to reproduce the issue

Play a Dolby Digital Plus 7.1 channels track within a mkv file

### Expected behavior

We should listen all channels and have a smooth and normal playback.

With Samsung Galaxy S21 (Android 12), there's no sound at all and movies stutters a lot

With Freebox Pop (Android 10 beta), there is some sound but the movie stutters a lot as well

### Your phone/tablet/androidTV model

Freebox Pop & Samsung Galaxy S21

### Operating system version

Android 10 beta (Freebox) & Android 12 (S21)

### Application version and app store

6.0.75 Google Play

**Comment by courville** [comment]
Can you please share a sample showing the issue?

**Comment by mitch133** [comment]
Sure, here you have

https://we.tl/t-y87I0NWt23

**Comment by courville** [comment]
Thanks. I guess there is indeed an issue with downmixing on nova. On your phone you can deactivate audio downmixing to stereo, it should bring sound back. I will investigate.

**Comment by mitch133** [comment]
Hello. Yes, deactivating audio downmixing solves the problem on my phone. But as I mentioned, no problem with AAC 7.1, even with downmixing active. This specifically happens with EAC3 from what I observed. 

On the other hand, it makes me think I reported 
here, some weeks ago, a similar problem with some EAC3 + ATMOS files. And more recently with DTS-ES 6.1 files. So downmixing may indeed be the issue, even if this doesn't occur in every case and with every terminal. Well, good luck with that!

**Comment by courville** [comment]
OK issue is present on nova-5_15_21 and nova-6_0_71 thus not a regression at least.

**Comment by mitch133** [comment]
Hi @courville , ok, good to know indeed.

Another thing I have to mention, is I installed yesterday the 6.0.76 version you sent us on my Freebox.
But today, I have the 6.0.75 (20221023.1335) version again from Google Store!!! How is it possible ? It's weird because I did absolutely nothing, and it's weird because the little stuttering I could notice with this same version before (on the Freebox Pop) is gone, even if I still face other issues.

Anyway, just for you to know here's another sample with "everything" in it (DTS-ES 6.1 + EAC3 7.1) and a good overview of my problems

https://we.tl/t-KDkN79DCl8

**1. In passthrough (mode 2) // Freebox Pop Android 10 beta**

- Sound and image are OK with DTS-ES 6.1
- No sound with EAC3 7.1 (due to SPDIF connection to my receiver not decoding this format) + strong stuttering

**2. In normal mode / no passthrough // Freebox Pop Android 10 beta**

- Strange piercing sounds with DTS-ES 6.1 + strong slowdown in fps
- Sound and image are OK with EAC3 7.1

**3. With Android 12 + Galaxy S21**

-  Sound and image are ok with DTS-ES 6.1, unless I check the audio / stereo mix box (in this case piercing sound and slowdown appear)
-  No sound + strong stuttering with EAC3 7.1, unless I check the audio / stereo mix box (in this case everything goes well, sound and image play normally)
